src/proc.h

```diff
@@ -192,6 +192,8 @@ class job_t {
     /// Sets the command.
     void set_command(const wcstring &cmd) { command_str = cmd; }
 
+    const wcstring preview() const { return processes.empty() ? L"" : processes[0]->argv0() + wcstring(L" ..."); }
```

Comment:
First `const` is unnecessary. Should be `wcstring preview() const { ...} 

Can you add a comment here describing what this is for?